src/commands/openWorkflowStepLogs.ts

```diff
@@ -0,0 +1,22 @@
+import * as vscode from "vscode";
+import {WorkflowStepNode} from "../treeViews/workflows/workflowStepNode";
+
+type WorkflowStepCommandArgs = Pick<WorkflowStepNode, "job" | "step" | "gitHubRepoContext">;
+
+export function registerOpenWorkflowStepLogs(context: vscode.ExtensionContext) {
+  context.subscriptions.push(
+    vscode.commands.registerCommand("github-actions.step.logs", async (args: WorkflowStepCommandArgs) => {
+      const job = args.job.job;
+      let url = job.html_url ?? "";
+      const stepName = args.step.name;
```

Comment:
First we get the job with the `html_url` which is the base URL for the run.  Then we are setting a local `stepName` so we can use that to find out what index number it is in an array.